### ISSUE TITLE
Accessibility with Nested and References samples

### ISSUE THREAD

**Issue by danizen** [mozilla-services/react-jsonschema-form#689]
### Description

Here in the federal government, we have to care a lot about accessibility, and not just for the completely blind.  I'm not quite sure how to make the Nested or References samples accessibly, particularly around tab order.

### Steps to Reproduce

1. Access the playground on github.io
2. Select the Nested example (or Refences example)
3. Use mouse to place Focus on first form field
4. Tab through the fields and check that (a) there is a visual indication of focus, (b) all inputs and buttons get focus.
5. Also check for keyboard equivalence on Nested to move things up and down.

#### Expected Behavior 

This should either be added to the component, or there should be guidelines on extending the component to provide accessibility short-cuts in the documentation.

#### Actual Behavior

Without a clue how to add this, I attempted to remedy this on my own by (a) forking, (b) cloning to locally development, (c) getting the playground running with npm install and npm start, and (d) copying the css and js of bootstrap accessibility plugin locally.   I didn't fool around with webpack, but built bootstrap-accessibility with grunt, then copied `plugins/css/bootstrap-accessibility.css` and `plugins/js/bootstrap -accessibility.min.js` into playground, and instrumented `devServer.js` to serve them and playground/index.html to reference them.

Since I don't see bootstrap's JS in `playground/index.html` or `package.json`, it wasn't clear to me whether this is the best way to make the component accessible.   I'll leave it to front-end experts.

### Version

0.49.0

**Comment by glasserc** [comment]
Hi, I also care about accessibility but I'm not really an accessibility expert, or honestly even a front-end expert. Currently we explicitly add `tabIndex="-1"` on the add element/move element up/move element down buttons, but I think that's more an accident of history than a design feature (see #392, #109). It wouldn't be difficult to remove `tabIndex="-1"` from the places where it is present. Would that be sufficient for your needs? I've pushed to the [playground](https://mozilla-services.github.io/react-jsonschema-form/) a version removing all `tabIndex` attributes if you want to give it a try.

> 5. Also check for keyboard equivalence on Nested to move things up and down.

Sorry, I didn't understand what this means. Is this just to reinforce that accessibility is broken on both `Nested` and `References` examples, or are you emphasizing that "move things up and down" buttons are not accessible, or is there some other "moving things up and down" that is relevant to accessibility that I'm not familiar with?

**Comment by glasserc** [comment]
This looks like it's stale, so I'm closing it.
